src/test/nexusTestUtils.js

```diff
@@ -52,104 +52,167 @@ fluid.test.nexus.assertContainsComponent = function (componentRoot, parentPath,
     jqUnit.assertValue(parentPath + " component contains " + childName, parent[childName]);
 };
 
+/**
+ * Assert that the object toTest contains a set of keys with given values.
+ * Can also be understood as "assert that the object expected is a subset of the object toTest".
```

Comment:
In fact we already have such an assertion method which is https://docs.fluidproject.org/infusion/development/jqUnit.html#jqunitassertlefthandmessage-expected-value